extensions/rewiser/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "rewiser",
+  "title": "Rewiser",
+  "description": "Track and manage your Rewiser transactions directly from Raycast",
+  "icon": "rewiserlogo.png",
+  "author": "rewiser_io",
+  "license": "MIT",
+  "keywords": [
+    "finance",
+    "expense",
+    "income",
+    "ai",
+    "natural language",
+    "transaction",
+    "money",
+    "budget",
+    "accounting"
+  ],
+  "categories": [
+    "Finance",
+    "Productivity"
+  ],
+  "commands": [
+    {
+      "name": "add-transaction",
+      "title": "Add Transaction",
+      "subtitle": "Rewiser",
+      "description": "Add a transaction by simply saying \"I spent $20 at a restaurant\"",
+      "mode": "view",
+      "command": "src/add-transaction.tsx"
```

Comment:
You don't need to specify the `command`, it should automatically fetch it based on the `name`.